test/chainrules.jl

```diff
@@ -212,8 +209,61 @@ using Zygote, Test, ChainRules
         @test (nothing,) == Zygote.gradient(x->not_diff_kw_eg(x, 2), 10.4)
         @test (nothing,) == Zygote.gradient(x->not_diff_kw_eg(x, 2; kw=2.0), 10.4)
     end
+
+    @testset "rrule_via_ad" begin
+        @testset "basic" begin
+            test_rrule(round, 2.2; rrule_f=rrule_via_ad)
```

Comment:
these tests need https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/166/files to be merged first